<!DOCTYPE html> <html lang="fr"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Gaëlle Lelandais</title> <meta name="author" content="Gaëlle Lelandais"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://gaellelelandais.github.io/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?6185d15ea1982787ad7f435576553d64"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">repositories</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">people</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/publications/">publications</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/projects/">projects</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/blog/">blog</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fa-solid fa-moon"></i> <i class="fa-solid fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Gaëlle</span> Lelandais </h1> <p class="desc">Généralités, informations pratiques, etc.</p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/prof_pic-480.webp 480w, /assets/img/prof_pic-800.webp 800w, /assets/img/prof_pic-1400.webp 1400w, " sizes="(min-width: 800px) 231.0px, (min-width: 576px) 30vw, 95vw" type="image/webp"></source> <img src="/assets/img/prof_pic.jpg?3c7cf38daed45e04d18461bce08bf8e9" class="img-fluid z-depth-1 rounded" width="100%" height="auto" alt="prof_pic.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <div class="more-info"> <p>Institut I2BC</p> <p>Bat. 23A, Lab. N1 - 119</p> <p>Gif-sur-Yvette</p> </div> </div> <div class="clearfix"> <h3 id="qui-suis-je-">Qui suis-je ?</h3> <p>Je suis Professeure à l’<a href="https://www.universite-paris-saclay.fr/" rel="external nofollow noopener" target="_blank">Université Paris Saclay</a>.</p> <p>J’enseigne <strong>les statistiques, la bioinformatique et l’analyse de données biologiques</strong> aux étudiantes et étudiants de Licence et de Master. Je suis également responsable de la <a href="https://www.universite-paris-saclay.fr/formation/licence-double-diplome/mathematiques-sciences-de-la-vie" rel="external nofollow noopener" target="_blank">Licence Double Diplôme (LDD) Mathématiques, Sciences de la Vie</a>. J’aime l’idée de partager, avec celles et ceux qui pourraient en avoir besoin, mes ressources pédagogiques. Vous trouverez ainsi <a href="https://gaellelelandais.github.io/teaching/">ici</a> des documents de cours que j’ai pu créer pour diverses formations. J’espère qu’ils seront utiles aux personnes qui pourraient les consulter.</p> <p>J’effectue ma recherche au sein de l’<a href="https://www.i2bc.paris-saclay.fr/" rel="external nofollow noopener" target="_blank">Institut de Biologie Intégrative de la Cellule (I2BC)</a> du CNRS. Je suis intéressée par les <strong>problématiques de Big Data pour les Sciences de la Vie</strong>. Au cours de mon parcours scientifique, j’ai travaillé dans des équipes de biologie expérimentale productrices de données, mais aussi dans des équipes de bioinformatique productrices de méthodologies d’analyses. J’ai dans ces contextes contribué à de <a href="https://gaellelelandais.github.io/projects/">multiples projets de recherche</a>.</p> <h3 id="pourquoi-ce-site-">Pourquoi ce site ?</h3> <p><strong>Ce site est mon espace numérique personnel</strong>. J’y partage des ressources et mes refléxions, en relation avec mes expériences d’enseignante et de chercheuse à l’Université. Son contenu n’engage pas les institutions pour lesquelles je travaille. L’envoi de tout commentaire, remarque ou demande de complément d’information est encouragé. N’hésitez pas à m’écrire ou m’interpeller via les espaces de discussions. Au plaisir donc, d’échanger avec vous !</p> </div> <h2><a href="/news/" style="color: inherit;">news</a></h2> <div class="news"> <div class="table-responsive" style="max-height: 60vw"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Feb 27, 2024</th> <td> En septembre dernier, j’ai suivi la formation <a href="https://www.universite-paris-saclay.fr/actualites/un-module-de-formation-pour-combattre-les-violences-sexistes-et-sexuelles" rel="external nofollow noopener" target="_blank">Agir contre les violences sexistes et sexuelles</a>. J’ai trouvé cette formation très bien réalisée et très utile pour des prises de conscience importantes. Je recommande à toutes les personnes qui travaillent dans le contexte de notre Université de prendre le temps de la suivre et pour donner l’exemple, je partage ici la certification que j’ai obtenue : <a href="../assets/pdf/paris-saclay-vss-certificat-Gae%CC%88lle-LELANDAIS.pdf">fichier PDF</a>. </td> </tr> <tr> <th scope="row" style="width: 20%">Feb 2, 2024</th> <td> Hier, j’ai témoigné de mon parcours de vie (pro et perso) dans le cadre de la formation “Oser” du CEA. Un moment d’échanges très riche entre femmes. Merci à <a href="https://www.linkedin.com/in/carolinemariage/" rel="external nofollow noopener" target="_blank">Caroline Mariage</a> pour l’animation et <a href="https://www.linkedin.com/in/vivienne-fardeau-847b2862/" rel="external nofollow noopener" target="_blank">Vivienne Fardeau</a> pour l’invitation et l’organisation. </td> </tr> <tr> <th scope="row" style="width: 20%">Jan 30, 2024</th> <td> Samedi 03 Février aura lieu les <a href="https://www.universite-paris-saclay.fr/actualites/journees-portes-ouvertes-de-luniversite-paris-saclay-2024" rel="external nofollow noopener" target="_blank">Journées Portes Ouvertes</a> de l’Université Paris Saclay. Je serai présente sur le stand de la formation <a href="https://www.universite-paris-saclay.fr/formation/licence-double-diplome/mathematiques-sciences-de-la-vie" rel="external nofollow noopener" target="_blank">LDD Mathématiques, Sciences de la Vie</a>. Cette journée sera également l’occasion <a href="../assets/pdf/2024-01-30_flyer_JPO.pdf">d’un moment de recontre</a>, entre les élèves actuellement dans la formation et celles et ceux en sont diplomé(e)s. Merci à Nerea Braseo pour la conception du flyer <img class="emoji" title=":sunflower:" alt=":sunflower:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f33b.png" height="20" width="20">. J’ai hâte d’y être ! </td> </tr> <tr> <th scope="row" style="width: 20%">Jan 30, 2024</th> <td> Le <a href="../assets/pdf/2024-01-30_PhD_Track.pdf">projet d’Anakim Gualdoni</a> a été sélectionné dans le cadre du programme d’accompagnement vers le doctorat de l’Université Paris Saclay <a href="https://www.universite-paris-saclay.fr/phd-track-programme-daccompagnement-vers-le-doctorat" rel="external nofollow noopener" target="_blank">PhD Track</a>. Je suis ravie d’être sa “mentore” pour ce projet ! Bravo Anakim <img class="emoji" title=":clap:" alt=":clap:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f44f.png" height="20" width="20">, vivement la suite <img class="emoji" title=":blush:" alt=":blush:" src="https://github.githubassets.com/images/icons/emoji/unicode/1f60a.png" height="20" width="20">. </td> </tr> <tr> <th scope="row" style="width: 20%">Jan 9, 2024</th> <td> Je présente aujourd’hui l’article <a href="https://academic.oup.com/nargab/article/5/4/lqad104/7458894" rel="external nofollow noopener" target="_blank"><strong>3D models of fungal chromosomes to enhance visual integration of omics data</strong></a>, devant les membres du conseil scientifique du <a href="https://www.i2bc.paris-saclay.fr/disciplinary-department/genome-biology-department/" rel="external nofollow noopener" target="_blank">Département Biologie des Génomes</a> de l’I2BC : <a href="../assets/pdf/2024-01-09_highligths_GL.pdf">fichier PDF de la présentation</a>. </td> </tr> </table> </div> </div> <h2><a href="/blog/" style="color: inherit;">latest posts</a></h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row" style="width: 20%">Jan 15, 2024</th> <td> <a class="news-title" href="/blog/2024/apprenante-4semaines/">Apprenante pendant 4 semaines, retour d’expérience(s)</a> </td> </tr> <tr> <th scope="row" style="width: 20%">Jan 11, 2024</th> <td> <a class="news-title" href="/blog/2024/plaisir-ecrire/">Ecrire, juste pour le plaisir</a> </td> </tr> </table> </div> </div> <h2><a href="/publications/" style="color: inherit;">selected publications</a></h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/Neurospora_crassa_WT_50kb-480.webp 480w, /assets/img/publication_preview/Neurospora_crassa_WT_50kb-800.webp 800w, /assets/img/publication_preview/Neurospora_crassa_WT_50kb-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/Neurospora_crassa_WT_50kb.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Neurospora_crassa_WT_50kb.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="10.1093/nargab/lqad104" class="col-sm-8"> <div class="title">3D models of fungal chromosomes to enhance visual integration of omics data</div> <div class="author"> Thibault Poinsignon, Mélina Gallopin, Pierre Grognet, Fabienne Malagnac, <em>Gaëlle Lelandais</em>, and Pierre Poulain</div> <div class="periodical"> <em>NAR Genomics and Bioinformatics</em>, Dec 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/lqad104.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>The functions of eukaryotic chromosomes and their spatial architecture in the nucleus are reciprocally dependent. Hi-C experiments are routinely used to study chromosome 3D organization by probing chromatin interactions. Standard representation of the data has relied on contact maps that show the frequency of interactions between parts of the genome. In parallel, it has become easier to build 3D models of the entire genome based on the same Hi-C data, and thus benefit from the methodology and visualization tools developed for structural biology. 3D modeling of entire genomes leverages the understanding of their spatial organization. However, this opportunity for original and insightful modeling is underexploited. In this paper, we show how seeing the spatial organization of chromosomes can bring new perspectives to omics data integration. We assembled state-of-the-art tools into a workflow that goes from Hi-C raw data to fully annotated 3D models and we re-analysed public omics datasets available for three fungal species. Besides the well-described properties of the spatial organization of their chromosomes (Rabl conformation, hypercoiling and chromosome territories), our results highlighted (i) in Saccharomyces cerevisiae, the backbones of the cohesin anchor regions, which were aligned all along the chromosomes, (ii) in Schizosaccharomyces pombe, the oscillations of the coiling of chromosome arms throughout the cell cycle and (iii) in Neurospora crassa, the massive relocalization of histone marks in mutants of heterochromatin regulators. 3D modeling of the chromosomes brings new opportunities for visual integration of omics data. This holistic perspective supports intuition and lays the foundation for building new concepts.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">10.1093/nargab/lqad104</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Poinsignon, Thibault and Gallopin, Mélina and Grognet, Pierre and Malagnac, Fabienne and Lelandais, Gaëlle and Poulain, Pierre}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{{3D models of fungal chromosomes to enhance visual integration of omics data}}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{NAR Genomics and Bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{lqad104}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">dec</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2631-9268}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/nargab/lqad104}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1093/nargab/lqad104}</span><span class="p">,</span>
  <span class="na">eprint</span> <span class="p">=</span> <span class="s">{https://academic.oup.com/nargab/article-pdf/5/4/lqad104/54014196/lqad104.pdf}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/978-1-0716-2257-5.webp-480.webp 480w, /assets/img/publication_preview/978-1-0716-2257-5.webp-800.webp 800w, /assets/img/publication_preview/978-1-0716-2257-5.webp-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/978-1-0716-2257-5.webp" class="preview z-depth-1 rounded" width="100%" height="auto" alt="978-1-0716-2257-5.webp" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Denecker2022" class="col-sm-8"> <div class="title">Omics Analyses: How to Navigate Through a Constant Data Deluge</div> <div class="author"> Thomas Denecker, and <em>Gaëlle Lelandais</em> </div> <div class="periodical"> Dec 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/978-1-0716-2257-5_25.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Omics data are very valuable for researchers in biology, but the work required to develop a solid expertise in their analysis contrasts with the rapidity with which the omics technologies evolve. Data accumulate in public databases, and despite significant advances in bioinformatics softwares to integrate them, data analysis remains a burden for those who perform experiments. Beyond the issue of dealing with a very large number of results, we believe that working with omics data requires a change in the way scientific problems are solved. In this chapter, we explain pitfalls and tips we found during our functional genomics projects in yeasts. Our main lesson is that, if applying a protocol does not guarantee a successful project, following simple rules can help to become strategic and intentional, thus avoiding an endless drift into an ocean of possibilities.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inbook</span><span class="p">{</span><span class="nl">Denecker2022</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Denecker, Thomas and Lelandais, Ga{\"e}lle}</span><span class="p">,</span>
  <span class="na">editor</span> <span class="p">=</span> <span class="s">{Devaux, Fr{\'e}d{\'e}ric}</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Omics Analyses: How to Navigate Through a Constant Data Deluge}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Yeast Functional Genomics: Methods and Protocols }</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer US}</span><span class="p">,</span>
  <span class="na">address</span> <span class="p">=</span> <span class="s">{New York, NY}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{457--471}</span><span class="p">,</span>
  <span class="na">isbn</span> <span class="p">=</span> <span class="s">{978-1-0716-2257-5}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-1-0716-2257-5_25}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://doi.org/10.1007/978-1-0716-2257-5_25}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" srcset=" /assets/img/publication_preview/m_cover-480.webp 480w, /assets/img/publication_preview/m_cover-800.webp 800w, /assets/img/publication_preview/m_cover-1400.webp 1400w, " sizes="200px" type="image/webp"></source> <img src="/assets/img/publication_preview/m_cover.gif" class="preview z-depth-1 rounded" width="100%" height="auto" alt="m_cover.gif" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="Denecker2020" class="col-sm-8"> <div class="title">Functional networks of co-expressed genes to explore iron homeostasis processes in the pathogenic yeast Candida glabrata</div> <div class="author"> Thomas Denecker, Youfang Zhou Li, Cécile Fairhead, Karine Budin, Jean-Michel Camadro, Monique Bolotin-Fukuhara, Adela Angoulvant, and <em>Gaëlle Lelandais</em> </div> <div class="periodical"> <em>NAR Genomics and Bioinformatics</em>, Apr 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/pdf/lqaa027.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="abstract hidden"> <p>Candida glabrata is a cause of life-threatening invasive infections especially in elderly and immunocompromised patients. Part of human digestive and urogenital microbiota, C. glabrata faces varying iron availability, low during infection or high in digestive and urogenital tracts. To maintain its homeostasis, C. glabrata must get enough iron for essential cellular processes and resist toxic iron excess. The response of this pathogen to both depletion and lethal excess of iron at 30°C have been described in the literature using different strains and iron sources. However, adaptation to iron variations at 37°C, the human body temperature and to gentle overload, is poorly known. In this study, we performed transcriptomic experiments at 30°C and 37°C with low and high but sub-lethal ferrous concentrations. We identified iron responsive genes and clarified the potential effect of temperature on iron homeostasis. Our exploration of the datasets was facilitated by the inference of functional networks of co-expressed genes, which can be accessed through a web interface. Relying on stringent selection and independently of existing knowledge, we characterized a list of 214 genes as key elements of C. glabrata iron homeostasis and interesting candidates for medical applications.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">Denecker2020</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Functional networks of co-expressed genes to explore iron homeostasis processes in the pathogenic yeast Candida glabrata}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{2631-9268}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{http://dx.doi.org/10.1093/nargab/lqaa027}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/nargab/lqaa027}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{NAR Genomics and Bioinformatics}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Oxford University Press (OUP)}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Denecker, Thomas and Zhou Li, Youfang and Fairhead, Cécile and Budin, Karine and Camadro, Jean-Michel and Bolotin-Fukuhara, Monique and Angoulvant, Adela and Lelandais, Gaëlle}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">apr</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%67.%6C%65%6C%61%6E%64%61%69%73.%77%6F%72%6B@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fa-solid fa-envelope"></i></a> <a href="https://orcid.org/0000-0002-2842-6172" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=_lbAeHAAAAAJ&amp;hl" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://github.com/gaellelelandais" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-github"></i></a> <a href="https://www.linkedin.com/in/ga%C3%ABlle-lelandais-9458505" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fa-brands fa-linkedin"></i></a> <a href="/feed.xml" title="RSS Feed"><i class="fa-solid fa-square-rss"></i></a> </div> <div class="contact-note"> Pour me contacter, le plus simple de m'envoyer un email ou de m'écrire via Linkedin. </div> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Gaëlle Lelandais. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. Last updated: February 27, 2024. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?9b43d6e67ddc7c0855b1478ee4c48c2d" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>